charts/vela-core/templates/defwithtemplate/storage.yaml

```diff
@@ -126,14 +126,14 @@ spec:
         	// +patchKey=name
         	volumes: pvcVolumesList + configMapVolumesList + secretVolumesList + emptyDirVolumesList
 
-        	containers: [...{
+        	containers: [{
         		// +patchKey=name
         		env: configMapEnvMountsList + secretEnvMountsList
         		// +patchKey=name
         		volumeDevices: volumeDevicesList
         		// +patchKey=name
         		volumeMounts: pvcVolumeMountsList + configMapVolumeMountsList + secretVolumeMountsList + emptyDirVolumeMountsList
-        	}]
+        	}, ...]
```

Comment:
After this modification, only the first container can be patched, right?

Reply:
yes，only the main container will be patched, exclude sidecar and init container

Reply:
> After this modification, only the first container can be patched, right?

yes，only the main container will be patched, exclude sidecar and init container